features/cellular/framework/targets/QUECTEL/EC2X/QUECTEL_EC2X.h

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright (c) 2019, Arm Limited and affiliates.
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#ifndef QUECTEL_EC2X_H
+#define QUECTEL_EC2X_H
+
+#ifdef TARGET_FF_ARDUINO
```

Comment:
These should be defined in mbed_lib.json as target_overrides.

Reply:
Would argue to remove these. Not sure of their exact purpose. While defining pins in your mbed_app.json you can alwyas spell out an arduino pin name convention. It's no different than providing any other pin name, e.g., PE15 or D1, it's just a convention.

Reply:
> Would argue to remove these.

Why would we want to leave this in - setting tx/rx to Dx pins?

Reply:
Compilation error is enough and correct way to handle this